_specs/ecip-_.md

```diff
@@ -11,11 +11,23 @@ resolution: TBD
 
 ### Abstract
 
-Forking Meta ECIPs (defined as Meta ECIPs specifying any Standards-Core track ECIP and an activation block number) should be complete and unique.
+Forking Meta ECIPs (defined as Meta ECIPs _intended_ to specify any Standards-Core track ECIP and an activation block number) should contain only placeholder information
```

Comment:
We can actually give "Forking Meta ECIPs" a new category. A lot of people have pointed out that hard fork ECIPs does not nicely fit in "meta" category.

Reply:
Interesting idea. See #224.